libraries/wallet/include/graphene/wallet/wallet.hpp

```diff
@@ -1538,6 +1538,10 @@ class wallet_api
          
       order_book get_order_book( const string& base, const string& quote, unsigned limit = 50);
 
+      signed_transaction multisig_sign_transaction( signed_transaction tx,
```

Comment:
Please add a comment that documents the call and its parameters, see approve_proposal call a few lines above.